### DISCUSSION TITLE
Give chance to translate "Drop file or click to choose" label

### DISCUSSION THREAD

**Discussion by FabrizioInfante** [laravel/nova-issues#5188] [Ideas & Feature Requests] [answered]
Hello everyone! I have an app with Laravel Nova in italian and I'd like to translate "Drop file or click to choose":

<img width="941" alt="Screenshot 2022-12-28 alle 12 01 51" src="https://user-images.githubusercontent.com/520172/209802346-ba7d8854-ace9-4306-a8a4-3fb28045db87.png">

But I don't see the key in translations files. Code that generate this screenshot is:

Image::make('Foto', 'photo_path'),

**Comment by crynobone** [comment, marked as answer]
You can just add the line: `"Drop file or click to choose": "Drop file or click to choose...",`

**Reply by FabrizioInfante** [reply]
Thanks very much @crynobone!!!
